Upgrade deno_std to 0.3.4 and update tests.

nilsnh opened the pull request:
I ran the tests and they now pass. I have Prettier installed in my VSCode and it changed single quotes to double quotes I see. 

This should fix #1 .

nilsnh commented:
Woops, hmm. It works locally. 🤔

sholladay commented on dependencies.js:
```diff
@@ -1,3 +1,7 @@
-export * as http from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/mod.ts';
-export { Status as status } from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts';
-export { STATUS_TEXT as statusText } from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts';
+export * as http from "https://deno.land/x/http@61af419bbc5717c2e2552050aacb20ef1b17480b/server.ts";
```
I guess this is the line that fixes the README example, right? Any breaking changes I should be aware of?

nilsnh replied:
`deno_std` doesn't have a changelog. But [deno's changelog](https://github.com/denoland/deno/releases) for 0.3.4 mentions that the deno module is gone and replaced by a Deno global var. Other than that it's hard to say.

sholladay commented:
I fixed the formatting and tested using the README example, but it crashed for me.

I took the README example, modified the import of `main.js` to point to my local repo on this branch, and then put that in a file named `server.js`. When I run `deno server.js`, it crashes with:

```js
/Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:85:22
                throw new ProtocolError(`malformed MIME header line: ${str(kv)}`);
                      ^
Uncaught ProtocolError: malformed MIME header line: �   ��39/5
    at ProtocolError (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:20:5)
    at readMIMEHeader (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:95:15)
```

nilsnh commented:
Hi again, thanks for checking out the PR. I pulled down your updates and ran `deno --reload test.js`, and they pass. You might be on an older version of deno?

<img width="674" alt="image" src="https://user-images.githubusercontent.com/282865/56189261-c8c73400-6027-11e9-8d8c-323a7d048d6a.png">

sholladay commented:
The tests pass. However, the README example does not. That's what I described. :)

nilsnh commented:
Aha! Hmm. I got the README example running now by doing `import pogo from "./pogo/main.js";` I have the code in this pull request in a sub folder of the example.

So, the README example might be failing because it fetches code from the current master branch?

nilsnh commented:
Ah, 

> I fixed the formatting and tested using the README example, but it crashed for me.
> 
> I took the README example, modified the import of `main.js` to point to my local repo on this branch, and then put that in a file named `server.js`. When I run `deno server.js`, it crashes with:
> 
> ```js
> /Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:85:22
>                 throw new ProtocolError(`malformed MIME header line: ${str(kv)}`);
>                       ^
> Uncaught ProtocolError: malformed MIME header line: �   ��39/5
>     at ProtocolError (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:20:5)
>     at readMIMEHeader (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:95:15)
> ```

Hmm. Did you also try `deno --reload server.js`, could it be that Deno used some old cached dependencies?

sholladay commented:
I ended up tracking that error down to a problem with my machine. My browser was attempting to upgrade all requests to HTTPS on localhost due to the fact that I had used an app on localhost with [HSTS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) enabled. Deno doesn't support HTTPS last I checked and apparently that is the weird error it's spewing out at the moment. Using a different browser or clearing the HSTS cache are two easy solutions.

So, in short, not a problem with our code. :)

Thanks for the patch!
